---
title: MBOC Alberts 5th Edition
permalink: /alberts-5e/
---


# Molecular Biology of the Cell
* Fifth Edition (2008)
* Authors: Bruce Alberts, Alexander Johnson, Julian Lewis, Martin Raff, Keith Roberts, and Peter Walter

## Chapter 1: Cells and Genomes
* p. 20 - defintion of **homologs**: All related genes that result from duplication 
	* Duplicate genes that are conserved in two branches after speciation are called **orthologs**
	* Duplicate genes within a single genome are **paralogs**
	* Orthologs and paralogs are subsets of the larger containing set of homologs
* **Dehydration Synthesis** (aka Condensation Synthesis) vs. **Hydrolysis**. See Alberts 5e Fig. 2-64 and 2-65 on p. 85, Alberts 5e Fig. 2-19 on p. 57, and JH handwritten notes (8/22/2020)

## Chapter 2: Cell Chemistry and Biosynthesis
### 10 Steps of Glycolysis
* Glycolysis converts glucose to pyruvate in 10 steps. 
* For more details, see JH handwritten notes August 7-13, 2020,  Alberts p. 120-121 (Panel 2-8), and [this page](/glycolysis). 

## Chapter 3: Proteins
* See p. 127-129 to see tables and panels of categories of polar vs. nonpolar sidechain/amino acids, acidic vs. basic sidechains, etc.

## Chapter 4: Chromosome Structure and Genomes
* Avg. size of a human gene is ~27,000 bp including both introns and exons. However, avg size of genes is only 1300 bp when all the introns are thrown out.  (p. 206) 
* See other interesting stats on human genes in Table 4-1 on p. 206.
* [Shared synteny](https://en.wikipedia.org/wiki/Synteny#Shared_synteny) aka conserved synteny = not only are DNA sequences / genes conserved, but the exact order of these genes are conserved across species (p. 207). 
* Three specialized DNA sequences are special on the chromosome: (1) [origin of replication](https://en.wikipedia.org/wiki/Origin_of_replication); prokaryotes have 1 origin per chromosome while eukaryotes have multiple ori's; (2) [centromere](https://en.wikipedia.org/wiki/Centromere); (3) [telomere](https://en.wikipedia.org/wiki/Telomere).


## Chapter 5: DNA Replication and Repair
*  DNA structure on Panel 2-6 Alberts p. 116-17 and Figure 4-4 and 4-5 on p. 198-199. See also JH handwritten notes August 25-29, 2020.
* Mutation rates in DNA are surprisingly low across multiple genera. It is about 1 mutation in 10<sup>9</sup> in both *E. coli* and *C. elegans* aka 1 base pair change per 1 billion bp. In addition, more indirect methods in humans using fibrinopeptides, fibrinogen, and fibrin formation during blood clotting have also produced a similar 1 in a billion figure (p. 264-265).

### History of discoveries
* [DNA Polymerase I](https://en.wikipedia.org/wiki/DNA_polymerase_I) was discovered by Arthur Kornberg in 1956 ( p. 266)
* [Okazaki fragments](https://en.wikipedia.org/wiki/Okazaki_fragments) discovered in mid- to late-1960s in Japan. This along with other evidence disproved earlier hypothesis that DNA polymerases could act in both 5'&#8594; 3' and in the 3'&#8594; 5' direction.
* Proofreading mechanisms:
	1.  ***Delay* between initial base pair hydrogen bonding and permanent phosophodiester covalent bonding**. First, a free nucleotide base pairs with the template DNA strand via hydrogen bonding. Obviously, the correct nucleotide is energetically more favorable. However, DNA polymerase is like a right hand whose fingers must tighten for the next step: covalently creating the phosphodiester bond on the backbone. It's hard to do that with the wrong nucleotide. So even if the wrong nucleotide slips in for step 1a, it's hard to "permanently" lock it in with the "finger-closing step". (p. 269 and Fig 5-4b)
	1. [**Exonucleolytic proof-reading**](https://en.wikipedia.org/wiki/Exonuclease) that excise nucleotides dNMP (aka 2'-**d**eoxyribose **n**ucleotide **m**ono **p**hosphates) one at a time from the new daughter strand. The exonuclease is actually a separate catalytic site on DNA polymerase. DNA Poly "absolutely requires a previously formed base-paired 3'-OH end of the primer strand." If the wrong dNMP is at the end of the daughter strand, the exonuclease is engaged and excised in the opposite 3'&#8594; 5' direction from the Poly's regular growing 5'&#8594; 3' direction.  (See Fig. 5-8 and 5-9 on p. 270.)
	1. **Strand-Directed Mismatch Repair System** which is a post-replication complex that catches errors missed in the initial multi-enzyme replication machine. This system finds mismatched base-pairs between a template and daughter strand. How does it know which is the template and which is the new strand? 
		* In prokaryotes like E.coli, by looking for unmethlyated A's. In bacteria, new strands eventually get all the A's in the sequence *GATC* methylated. Fresh new daughter strands have not gotten their A's methylated so that is the strand that is corrected. (p. 276-277)
		* Eukaryotes rarely (never?) have their A's methylated. Fresh, new daughter strands are identified because they have frequent ss nicks where ligase has not performed it's function.

* see also Table 5-1 on p. 271 to see how the 3 different error correcting mechanisms of 5'&#8594; 3' polymerization, 3'&#8594; 5' exonuclease proofreading, and strand-directed mismatch repair together combine to provide a 1 error per 10<sup>9</sup> bp during DNA replication.


### DNA Replication
* Major enzymes involved:
	* Helicase to make the initial cut to start the replication fork
	* Primase to add short RNA primers as a starting point for DNA Polymerase
	* DNA Polymerase and associated Exonuclease catalytic sites
	* Ligase to join together short Okazaki fragments on the lagging strand
	* Topoisomerase I and II that moves ahead of the replication fork and relieving the coil tension created by unwinding the double helix. [Topo I](https://en.wikipedia.org/wiki/Type_I_topoisomerase) creates a single strand nick while [Topo II](https://en.wikipedia.org/wiki/Type_II_topoisomerase) is a  gate-folded enzyme that moves strands past each other with a controlled cut and rejoin of one strand. See also p. 278-280.
