---
title: Felsenstein 2004
permalink: /felsenstein-2004/
---

# [Inferring Phylogenies](https://books.google.com/books/about/Inferring_Phylogenies.html?id=GI6PQgAACAAJ)
* Author: Joseph Felsenstein
* Sinauer Associates, 2004
* [Box link](https://app.box.com/s/fnsgn47kt87zveup29wku75e56vbi7k7)

## Chapter 1: Parsimony methods
* One of the earliest methods of inferring phylogenies are parsimony methods. Edwards and Cavalli-Sforza (1963) declared that the preferred evolutionary tree is the one that involves 'the minimum net amount of evolution'.
* In other words, we seek the phylogeny within which the fewest events occur while reconstructing the series of evolutionary events.

### A simple example
* Assume we have 5 species, each of which ahs been scored for 6 characters. Each character can only have 2 states: 0 or 1. 
* An event only allows a change in a character state from 0 --> 1 or 1 -->0.
* The initial state at the root node can be either state 0 or state 1.
* To find the most parsimonious tree, we must have a way of calculating how many changes of state are needed on a given tree. Suppose that someone proposes the phylogeny shown in Figure 1.1 (p. 2). The dataset is small enough in this example such that one can just "eyeball" the best reconstruction of evolution.

## Chapter 2: Counting evolutionary changes
## Chapter 3: How many trees are there?
## Chapter 4: Finding the best tree by heuristic search
## Chapter 5: Finding the best tree by branch and bound
## Chapter 6: Ancestral states and branch lengths
## Chapter 7: Variants of parsimony
## Chapter 8: Compatibility
## Chapter 9: Statistical properties of parsimony
## Chapter 10: A digression on history and philosophy
## Chapter 11: Distance matrix methods
## Chapter 12: Quartets of species
## Chapter 13: Models of DNA evolution
## Chapter 14: Models of protein evolution
## Chapter 15: Restriction sites, RAPDs, AFLPs, and microsatellites
## Chapter 16: Likelihood methods
## Chapter 17: Hadamard methods
## Chapter 18: Bayesian inference of phylogenies
## Chapter 19: Testing models, trees, and clocks
## Chapter 20: Bootstrap, jackknife, and permutation tests
## Chapter 21: Paired-sites tests
## Chapter 22: Invariants
## Chapter 23: Brownian motion and gene frequencies
## Chapter 24: Quantitative characters
## Chapter 25: Comparative methods
## Chapter 26: Coalescent trees
## Chapter 27: Likelihood calculations on coalescents
## Chapter 28: Coalescents and species trees
## Chapter 29: Alignment, gene families, and genomics
## Chapter 30: Consensus trees and distances between trees
## Chapter 31: Biogeography, hosts, and parasites
## Chapter 32: Phylogenies and paleontology
## Chapter 33: Tests based on tree shape
## Chapter 34: Drawing trees
## Chapter 35: Phylogeny software
