---
title: Home
---

<!-- *See also*
* Notes on [general chemistry](/general-chemistry/)
* Notes on [organic chemistry](/organic-chemistry/)
-->

# Biology textbook
* *Life, The Science of Biology*, [Tenth Edition](amazon.com/Life-Science-Biology-Sadava-Hardcover-dp-B00MEYGNSS/dp/B00MEYGNSS) published 2014
* Authors: [David Sadava](https://www.cmc.edu/news/professor-and-biologist-david-sadava-to-retire), [David M. Hillis](https://sites.cns.utexas.edu/hillislab/lab-members), [H. Craig Heller](https://profiles.stanford.edu/h-craig-heller), and [May R. Berenbaum](https://en.wikipedia.org/wiki/May_Berenbaum)

## Chapter 2: Review of [General Chemistry](/general-chemistry/)

### Chapter 2.1: How does Atomic Structure Explain the Properties of Matter?
* Review of [atomic orbitals](https://en.wikipedia.org/wiki/Electron_shell) p. 24-25 
	* An atomic orbital can hold at most two electrons.
	* Remember:
		* First Shell: *1s<sup>2</sup>* = total of **2 electrons** in this shell. Only an s-orbital shape and 1 orbital.
		* Second Shell: *2s<sup>2</sup> + 2p<sup>6</sup>* = total of **8 electrons** in this shell. Two subshells. The *s* subshell has one spherical orbital in it. The *p* subshell has 3 dumbbell-shaped orbitals. Thus there are a total of 4 orbitals.
		* Third Shell: *3s<sup>2</sup> + 3p<sup>6</sup> + 3d<sup>10</sup>* = total of **18 electrons** in this shell. Three subshells: s, p, d. As expected, the s subshell contains one s orbital. The p subshell contains 3 p orbitals. The d subshell has 5 d orbitals. 1+3+5= **9 orbitals** which collectively hold a maximum of 18 electrons.
		* Fourth Shell: *4s<sup>2</sup> + 4p<sup>6</sup> + 4d<sup>10</sup> + 4f<sup>14</sup>* = total of **32 electrons** in this shell. 1+3+5+7 = **16 orbitals** which can hold a maximum of 32 electrons  
		* Fifth Shell: *5s<sup>2</sup> + 5p<sup>6</sup> + 5d<sup>10</sup> + 5f<sup>14</sup> + 5g<sup>18</sup>* = total of **50 electrons** in this shell.
	* Filling order is based on [the Aufbau principle](https://en.wikipedia.org/wiki/Aufbau_principle) aka the [Madelung energy ordering rule](https://en.wikipedia.org/wiki/Aufbau_principle#Madelung_energy_ordering_rule):
		* 1s, 2s, 2p, 3s, 3p, 4s, 3d, 4p, 5s, 4d, 5p, 6s, 4f, 5d, 6p, 7s, 5f, 6d, 7p, 8s, etc.

### Table 2.1 Chemical Bonds - p. 26

| Type of Bond              | Description                                                         | Bond Energy (kcal/mol) |
|---------------------------|---------------------------------------------------------------------|------------------------|
| Covalent bond             | Sharing of electron pairs                                           | 50-110                 |
| Ionic bond          | Example NaCl — electrons stolen to complete octets in valence shell to produce *+cations* and *-anions*   | 3-7                    |
| Hydrogen bond             | Attraction between bonded H<sup>+</sup> atom and electronegative atom. Important for [secondary structure within a protein molecule](https://en.wikipedia.org/wiki/Protein_secondary_structure) but also occurs between molecules   | 3-7                    |
| Hydrophobic interaction   | Nonpolar substances in presence of water (or other polar substrate) | 1-2                    |
| van der Waals interaction | Interaction of electrons of nonpolar substances                     | 1                      |


### Concept of Electronegativity - p. 28
* When two atoms of different elements are covalently bonded in a molecule, their electrons are not equally shared. So one of the positively charged nuclei may exert a greater attractive force on the shared electron pair than the other nucleus. This attractive force is called **electronegativity**.
* p. 28: atoms with similar electronegativity = **nonpolar covalent bond**. Atoms with different electronegativity produce a **polar covalent bond**.
* Extreme electronegativity = powerful positive proton nuclues which leads to ability to attract greater than "fair share" of shared electrons in a molecule
* Maps to having almost having enough electrons in valencce shell to complete s2p6 octet
* In general, most electronegative columns in the periodic table are Groups 16 & 17 because they have 6 & 7 electrons, respectively in their outer valence shell.
* Conversely, the least electronegative groups in the periodic table are Groups 1 & 2 which respectively have 1 & 2 electrons in their outermost shell and these are the easiest electrons to lose.
* At the limit, when superstrong-electronegative and superweakly-electronegative elements interact, they don't just form a hyper-polar covalent bond; instead actually turn into an Ionic Bond.
* good illustration of this in **MSJ** p. 288, Figure 8.5.
### Ionic Bonds - p. 28-29
* When 2 interacting atoms have a large enough difference in electronegativity, ions form as the more powerfully electronegative nucleus strips electrons away from the weakly electronegative (aka positive atom). 
* Example of sodium Na and chlorine Cl
	* Na has its 1st two shells filled (aka *1s<sup>2</sup>* + *2s<sup>2</sup> + 2p<sup>6</sup>*). Also, as a [Group I element](https://iupac.org/what-we-do/periodic-table-of-elements/), it also has an extra electron in the next subshell: 3s<sup>2</sup> aka the first electron to appear in Shell M. It is easy to lose this electron.
	* Meanwhile, Cl only has 7 electrons in its  *3s<sup>2</sup> + 3p<sup>6</sup>* subshells. But ideally it would "like" **8 electrons** to complete subshell *3p<sup>6</sup>*. 
	* So in solution, Cl's more electronegative (aka more positive) nucleus "steals" the sole electron in Na's outermost subshell 3s<sup>2</sup>.
	* This produces a **positive cation** Na<sup>+</sup> and a **negative anion**  Cl<sup>-</sup> .
* This also explains the octet rule for molecules formed by atoms where *s<sup>2</sup> and p<sup>6</sup>* subshells are concerned.

### Hydrogen bonds
* Salts and interactions with polar molecules - p. 29
* Good interactive page on [hemoglobin and primary, secondary, tertiary, and quartenary structure](https://www.labxchange.org/library/items/lb:LabXchange:99c39c9f:lx_simulation:1)

### Chapter 2.4: H<sub>2</sub>0 has special properties - p. 32-34
* **Electronegativity causes strongly polar covalent bonds.** Water has a tetrahedral shape with oxygen in the center, two of the tetrahedral vertices occupied by hydrogen atoms, and the two remaining vertices occupied by an unbonded electron pair. Because oxygen has a high electronegativity relative to hydrogen, there is a  partial *delta-positive* charge on each hydrogen. Conversely, there is a partial *delta-negative* charge on each unbonded electron pair on the other two vertices. 
* Highly polar bonds then cause multiple and powerful hydrogen bonds between water molecules. Can also refer to these hydrogen bonds as intermolecular forces.
	* In a liquid state, water molecules are connected to each other by chains of powerful but intermittent hydrogen bonds. Special properties of liquid water thanks to multiple strings of hydrogen bonds:
		* High surface tension
		* High cohesion (water to water) and high adhesion (water to other substances) which are critical to [upward water flow in plant xylem](https://en.wikipedia.org/wiki/Xylem#Main_function_–_upwards_water_transport) 
		* High surface tension (water striders) and allows beads of water droplets (ants carrying a ball of water)
		* Lower vapor pressure (aka evaporation)
		* High specific heat
		* High boiling point aka heat of vaporization
	* When water freezes and turns into a solid, the daisy chain of randomly appearing/disappearing hydrogen bonds form into more permanent hydrogen bonds. These new hydrogen bonds follow from the partial delta-charges; the result is a  "hollow" crystal structure for ice which is why is less dense than liquid water. Each water molecule is connected to [4 other water molecules in a 3-d tetrahedral structure](https://www.youtube.com/watch?v=BxCnDJSE1ck&t=157s). 

### Review of Acids, Bases, and Buffers - p. 33-36


## Chapter 3: Macromolecules I - Proteins, Carbohydrates, Lipids (p. 39-61)
* Four kinds of macromolecules: proteins, carbohydrates, lipids, and nucleic acids
* Macromolecular polymers are made of different types of monomers:
	* Proteins are made up of **amino acid** monomers.
	* Carbohydrates are made up of simple **sugar** monomers.
	* Lipids are made up of lipid monomers connected by *noncovalent bonds*
* In general, only polymers with molecular weights > 1000 are considered macromolecules.
* Figure 3.1 (p. 40) has a good diagram of various functional groups, categories of compounds, and example compounds.
* Figure 3.3 (p. 41) has good % breakdown of macromolecules in living tissue.
* Reactants Monomer<sub>1</sub> + ... + Monomer<sub>n</sub> --> Polymer via removal of H<sub>2</sub>0 and addition of energy (endothermic reaction). Also called **condensation** or **dehydration reaction**.
* Reverse reaction of Reactant Polymer ---> breaking into smaller monomers by adding water and releasing energy from the reactant  (exothermic reaction). Called **hydrolysis**.

### Proteins
* The protein alphabet of monomers cosists of 20 amino acids. All proteins consist of a permutation of these 20 elemental units. 
	* The smaller limit of proteins is around **51 amino acids** for *[insulin](https://en.wikipedia.org/wiki/Insulin)* with molecular weight of 5,733.
	* The upper limit of protein chain size is around **27,000 amino acid** for the muscle protein *[titin](https://en.wikipedia.org/wiki/Titin)* with a molecular weight of almost 3 million.
* Polypeptide chains are linear (*aka* unbranched) sequences of covalently linked amino acids.
* Amino acids have 3 functional groups plus 1 hydrogen surrounding the central *&#945;* carbon:
	1. **carboxyl functional group** -COOH ionizes to -COO<sup>-</sup>
	1.  **amine group** -NH<sub>2</sub> ionizes to -NH<sub>3</sub><sup>+</sup>
	1. **R side chain**

**Proteins and associated functions**

| Category                 | Function                                                    |
|--------------------------|-------------------------------------------------------------|
| Enzymes                  | Catalyze biochemical reactions                              |
| Structural proteins      | Provide physical stability and movement                     |
| Defensive proteins       | Recognize and react to nonself substances (e.g. antibodies) |
| Signaling proteins       | Control physiological processes (e.g. hormones)             |
| Receptor proteins        | Receive and respond to chemical signals                     |
| Membrane transporters    | Regulate passage of substances across cell membranes        |
| Storage proteins         | Store amino acids for later use                             |
| Transport proteins       | Bind and carry substances within the organism               |
| Gene regulatory proteins | Determine rate of expression of genes                       |


## Chapter 4: Macromolecules II - Nucleic Acids (p. 62-76)
*







---
# Notes on MIT 7.00x at EdX
* *Course started: July 14, 2020*

## Lecture 1: Introduction and the Biochemistry of Life
* Seems like intro lectures by Eric Lander were recorded in the fall of 2012, because he references this [NYT Sunday Magazine article on pain](https://www.nytimes.com/2012/11/18/magazine/ashlyn-blocker-feels-no-pain.html) and [this drug for cystic fibrosis](https://www.nytimes.com/2012/02/01/business/fda-approves-cystic-fibrosis-drug.html). Also, he refers to an early lecture earlier that morning at the Broad Institute (probably by [Feng Zhang](https://www.broadinstitute.org/what-broad/areas-focus/project-spotlight/crispr-timeline) or a close collaborator) about work that was not yet published and about new developments in applying CRISPR/Cas9 to precisely edit DNA in mammalian systems.
* Timeline
	* **4.5 billion** years ago: Earth formed
	* **4.0 B** ya: Earth cooled enough for life
	* **3.7 B** ya: first evidence of prokaryotic life
	* **1.5 B** ya: first eukaryotic life with nucleus
	* **500 mya**: first multicellular life just prior to Cambrian Explosion
	* **5 mya**: first hominids aka Australopithecina or Hominina subtribe within the Hominidae Family.
	* **100,000 ya**: first homo sapiens
	* 1861 AD: MIT founded
* Fundamental principles and intellectual unification of biology
	* Biochemistry in ~1900: purification and analysis of unique proteins, nucleic acids, etc.
	* Genetics in ~1900: Organism minus one component, aka mutant. This component which they called a gene.
	* 1953: structure of DNA discovered by Watson & Crick, combining the two streams of biochem and genetics into the new unified science of **molecular biology**.
	* 1970s: Invention of **recombinant DNA** to isolate specific gene, read what protein it encodes. Take a protein and decide which gene codes for it.
	* 1980s: How do we map the genes that map to specific disease? Rather than studying single genes in isolation, was it possible to study all the genes for an organism at once? Yes, the beginning of **genomics**.

---

# Miscellaneous notes

* Good [YouTube video](https://youtu.be/ZesFyuFj30s) explaining how to use free online MolView.org. 
* Awesome [before and after video](https://twitter.com/DrewCoffman/status/1274743473732632576) showing off the special effects a single artist can create in Blender. Higher res version on [Youtube](https://youtu.be/FFJ_THGj72U)

## Different versions of 7.01 Intro Bio (course at MIT)

[Fundamentals of Biology](https://ocw.mit.edu/courses/biology/7-01sc-fundamentals-of-biology-fall-2011/syllabus/) was designed specifically for independent study and OpenCourseWare. It draws upon material developed for the three versions of MIT's Introductory Biology classes known as 7.012, 7.013, and 7.014. All three classes cover the same core material, which includes the fundamental principles of biochemistry, genetics, molecular biology, and cell biology. In addition, each version of the subject has its own distinctive material.

According to [this page](https://ocw.mit.edu/courses/biology/7-01sc-fundamentals-of-biology-fall-2011/), these courses were recorded in the following years

* [Fall, 2004](https://ocw.mit.edu/courses/biology/7-01sc-fundamentals-of-biology-fall-2011/index.htm) - **7.012** with Professors Eric Lander and Robert Weinberg, focuses on genomics, neurobiology, and cancer cell biology.  
* [Spring, 2005](https://ocw.mit.edu/courses/biology/7-014-introductory-biology-spring-2005/) - **7.014**, with Professors Graham Walker and Penny Chisholm, focuses on ecology and environment.
* **7.013** with Professors Hazel Sive and Tyler Jacks, focuses on development and cancer cell biology has been recorded twice:
	* [Spring 2018](https://ocw.mit.edu/courses/biology/7-013-introductory-biology-spring-2018/)  *(The Savada 10th Ed textbook I have is for this version of 7.013)*
	* [Spring 2013](https://ocw.mit.edu/courses/biology/7-013-introductory-biology-spring-2013/) 
