---
title: Home
---

### Different versions of Intro Bio course at MIT

There is a special version of the MIT intro bio course that was optimized for edX called [7.00x, Introduction to Biology - The Secret of Life](https://www.edx.org/course/introduction-to-biology-the-secret-of-life-3) taught by [Lander](https://www.broadinstitute.org/bios/eric-s-lander). Current lectures seem to have been recorded in 2012 with updates circa 2015.

Separately, [7.01 Fundamentals of Biology](https://ocw.mit.edu/courses/biology/7-01sc-fundamentals-of-biology-fall-2011/syllabus/) was designed specifically for OpenCourseWare. It draws upon material developed for the three versions of MIT's Introductory Biology classes known as 7.012, 7.013, and 7.014. All three classes cover the same core material, which includes the fundamental principles of biochemistry, genetics, molecular biology, and cell biology. In addition, each version of the subject has its own distinctive material.

According to [this page](https://ocw.mit.edu/courses/biology/7-01sc-fundamentals-of-biology-fall-2011/), these versions of 7.01 were recorded in the following years:

* [Fall, 2004](https://ocw.mit.edu/courses/biology/7-01sc-fundamentals-of-biology-fall-2011/index.htm) - **7.012** with Professors Eric Lander and Robert Weinberg, focuses on genomics, neurobiology, and cancer cell biology.  
* [Spring, 2005](https://ocw.mit.edu/courses/biology/7-014-introductory-biology-spring-2005/) - **7.014**, with Professors Graham Walker and Penny Chisholm, focuses on ecology and environment.
* **7.013** with Professors Hazel Sive and Tyler Jacks, focuses on development and cancer cell biology has been recorded twice:
	* [Spring 2018](https://ocw.mit.edu/courses/biology/7-013-introductory-biology-spring-2018/)  *(The Savada 10th Ed textbook I have is for this version of 7.013)*
	* [Spring 2013](https://ocw.mit.edu/courses/biology/7-013-introductory-biology-spring-2013/) 


### Species count as of 2014 - Sadava, p. 8

| Taxon    | Currently described species | *Estimated* number of living species |
|----------|---------------------------------------|------------------------------------------|
| Bacteria | 10,000                                | Millions                                 |
| Archea   | 300                                   | 1k - 1M                                  |
| Protists | 80,000                                | 500k - 1M                                |
| Plants   | 270,000                               | 400-500k                                 |
| Fungi    | 100,000                               | 1-2M                                     |
| Animals  | 1,300,000                             | 10M - 100M                               |

---
# Notes on MIT 7.00x at edX
*Course started: July 14, 2020*

## Lecture 1: Introduction and the Biochemistry of Life
### July 14 - 20, 2020
* Lectures by Eric Lander were likely recorded in the 2012 - 2013 academic year.
	* This is suggested by Lander's reference to this [NYT Sunday Magazine article on pain](https://www.nytimes.com/2012/11/18/magazine/ashlyn-blocker-feels-no-pain.html) which had just come out the "last weekend". He also discusses [this drug for cystic fibrosis](https://www.nytimes.com/2012/02/01/business/fda-approves-cystic-fibrosis-drug.html).
	* But most interestingly, Lander referred to a seminar he'd had at the [Broad Institute](https://en.wikipedia.org/wiki/Broad_Institute) just earlier that morning. He said they discussed a paper that had just been published  four weeks earlier; probably the seminal [2012 CRISPR/Cas9 paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6286148/pdf/nihms-995853.pdf) by Jinek, Chylinski, Fonfara, Hauer, Doudna, and Charpentier.
	* Of course, [Feng Zhang's lab](https://www.broadinstitute.org/what-broad/areas-focus/project-spotlight/crispr-timeline) then went on to publish this [2013 Science paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3795411/pdf/nihms444192.pdf) applying CRISPR/Cas9 to human and mouse cells.
* Timeline
	* **4.5 billion** years ago: Earth formed
	* **4.0 B** ya: Earth cooled enough for life
	* **3.7 B** ya: first evidence of prokaryotic life
	* **1.5 B** ya: first eukaryotic life with nucleus
	* **500 mya**: first multicellular life just prior to Cambrian Explosion
	* **5 mya**: first hominids aka Australopithecina or Hominina subtribe within the Hominidae Family.
	* **100,000 ya**: first homo sapiens
	* 1861 AD: MIT founded
* Fundamental principles and intellectual unification of biology
	* Biochemistry in ~1900: purification and analysis of unique proteins, nucleic acids, etc.
	* Genetics in ~1900: Organism minus one component, aka mutant. This component which they called a gene.
	* 1953: structure of DNA discovered by Watson & Crick, combining the two streams of biochem and genetics into the new unified science of **molecular biology**.
	* 1970s: Invention of **recombinant DNA** to isolate specific gene, read what protein it encodes. Take a protein and decide which gene codes for it.
	* 1980s: How do we map the genes that map to specific disease? Rather than studying single genes in isolation, was it possible to study all the genes for an organism at once? Yes, the beginning of **genomics**.

## Lecture 2: Biochemistry
### July 21 - 28, 2020
* Eduard Buchner and the founding of biochemistry. 
	* During a [series of experimnts from 1857-1877](https://en.wikipedia.org/wiki/Fermentation#History_of_the_use_of_fermentation), Louis Pasteur proved that microorganisms were involved in fermentation (conversion of juice into CO<sub>2</sub> and alcohol). This seemed to be a defense of vitalism.
	* However, [in 1897](https://en.wikipedia.org/wiki/Eduard_Buchner#Research), Buchner showed that if one took a mass of yeast and crushed all the living cells to death, one could extract dead "yeast juice" that could still perform fermentation. This showed that a nonliving enzyme ([zymaseproduced](https://en.wikipedia.org/wiki/Zymase)) by yeast was responsible for fermentation. 
* Composition of cells; covalent bonds
	* Atomic composition of human life: Consists largely of 63% H, 24% O, 10% C, 14% N, 0.2% phosphorus,  and less than 0.1% sulfur.
	* Molecular composition of human life measured by dry weight: 80% H<sub>2</sub>O. The remaining 20% can be subdivided into:
		* 50% protein
		* 15% carbohydrate
		* 15% nucleic acids
		* 10% lipids and fats
	* Covalent bonds for 6 most important elements: **C,H,O** and **N, P, Sulfur**. Number of bonds formed for an octet satisifed atom under biological conditions: 
		* Carbon = 4
		* Hydrogen = 1
		* Oxygen = 2
		* **Nitrogen = 3 or 4**
		* **Phosphorus = 5**
		* **Sulfur = sometimes 2** but up to **6 bonds**
* Noncovalent bonds, including hydrogen bonds (5 kcal/mol), ionic bonds, Van Der Waals forces
* Lipids and phospholipids
	* Oil and water don't mix b/c the polar water gets "offended" by the nonpolar interlopers interrupting the "desired" hydrogen bonding between water molecules
	* A nonpolar hydrocarbon (like cyclohexane C<sub>6</sub>H<sub>14</sub>) can be turned polar by converting them to:
		* Alcohols have a hydroxl -OH group attached.
		* Carboxylic Acids contain one or more -COOH groups.
	*  Fatty acids are a type of carboxylic acid. If you attach 3 of them to an alcolhol like glycerol, we create a "Tri-acyl-glyceride", aka triglyceride. This is a lipid
	* Lipids are [amphipathic](https://en.wikipedia.org/wiki/Amphiphile) with a hydrophlic head and hydrophobic tail.
	* When placed in a polar solvent like water, lipids spontaneously form globular structures like [micelles](https://en.wikipedia.org/wiki/Micelle) (monolayer with hydrophilic heads out, tails in) or [liposomes (bilayered)](https://en.wikipedia.org/wiki/Liposome). See also <a Href="https://bio.libretexts.org/Bookshelves/Biochemistry/Book%3A_Biochemistry_Online_(Jakubowski)/01%3A_LIPID_STRUCTURE/1.2%3A_Lipids_in_Water/1.2A%3A_Micelles_and_Bilayers">this page</a>
* ATP - adenosine triphosophate
* Carbohydrates
	* glucose = monosaccharide. Can form disaccharides, polysaccharides, etc.
* Remember, here are the types of forces that Lander emphasizes:
	1. **Ionic Bonds**
	2. **Covalent Bonds** (polar like O-H, N-H, C=O, vs. nonpolar like C-H, C-C, C=C, etc.)
	3. **Hydrogen Bonds** (O----H---O,  N----H---O ,  etc.)
	4. **Hydrophobic forces** that force hydrophobic molecules together, so that the liquid H<sub>2</sub>O molecules around them can form hydrogen bonds easily "as desired". 
	1. **Van Der Waals Forces** which are temporary charges that can add up to a lot, e.g,  geckos use these to climb walls


## Lecture 3: Proteins
### July 25 - 31, 2020
* Relatively constant planes in peptide with pivot points at the alpha carbons aka the carbons that the -R sidechains hang off of. See also Alberts p. 127
* Subcategories of the 20 amino acids
	* **Nonpolar and Noncharged (hydrophobic)**: Alanine, Valine, Methionine (with a sulfur), Leucine, Isoleucine, Phenylalanine, Tyrosine, Tryptophan
	* **Polar and Uncharged**: Serine, Threonine, Aspargine, Glutamine
	* **Polar, negatively charged**: Aspartic acid, glutamic acid
	* **Polar, positively charged**: lysine, arginine, histidine
	
	* **Special**:
		* Glycine very flexible, simple CH<sub>3</sub>
		* Proline (ring that bonds with the amino group)-- technically an imino acid, not an amino acid.
		* Cysteine with sulfur at the end. Can form disulfide bond.
	* Secondary structure
		* Alpha-helix. 3.7 amino acids per 360-degree turn of the alpha helix.
		* Beta-sheets
		* All other is called "loops" or "random coils"
	* Tertiary structure and quartenary structure is determined in part by the following types of interactions between sidechain groups:
		* Disulfide bonds (esp. cysteine side chain)
		* Ionic bonds between side chains
		* Hydrogen bonds between side chains
		* Van der Waals forces between side chains
	* X-Ray Protein Structure supplemental lecture by x-ray crystallographer Brian, a  scientist at MIT:
		* Metaphor of throwing basketballs and ping pong balls, first at a single parked car. Then, dropping many of these balls onto an entire parking lot full of regularly spaced parked cars.
		* To prepare them for scanning, proteins are first purified and then crystallized so they are in a regular repeating pattern (like in an orderly parking lot)
		* He uses the example of lysozym, an enzyme used to break down bacterial walls

## Lecture 4: Enzymes - Channel proteins, &#946;-barrels, TIM, flu burglar
### August 1, 2020
* Channel proteins
	* Design guideline: allow molecules of about 600 molecular weight, polar but no net charge (either positive or negative)
	* Solution: **OmpF channel protein**. (No wiki article available but see [related OmpA article](https://en.wikipedia.org/wiki/OmpA-like_transmembrane_domain).)
		* "residue" = synonym for side chain
		* has charged residues on the inside to repel potential passengers who are positively charged or negatively charged. The fact that these residues are polar will also keep out nonpolar passengers which we do not want to pass
		* Has hydrophobic sidechains along the exterior of the barrel so that it embedds properly with the hydrophobic C-H tails inside the phospholipid bilayer
	* Has a lot of charged residues
* Enzyme: [Triose Phosphate Isomerase](https://en.wikipedia.org/wiki/Triosephosphate_isomerase) *aka* TIM *aka* TPI
	* See Jeremy Knowles and W.J. Albery [TIM paper (1977)](https://pubs.acs.org/doi/10.1021/ar50112a001)
	* TIM interconverts between two isomers of triose phosphate
	* Essential to process of glycolysis in muscles for fast efficient energy production
	* See also [Protopedia article](https://proteopedia.org/wiki/index.php/Triosephosphate_Isomerase)
	* Convert G3P aka (glyceraldehyde 3-phosphate) to (DHAP) dihydroxyacetone phosphate via intermediate transition state (cis-enediol) which without TIM would take 26 kcal/mol
	* TIM stabilizes the transition state and also prevents the side-reaction of G3P losing it's phosophate group which tends to happen in water.
	* TIM increases speed of reaction by 10<sup>10</sup>. Aka, the difference in length of time between ~1 second and 300 years!
	* TIM is about 150x the size of reactant G3P and product DHAP. 
	* Three strategies used by TIM/TPI:
		1. Hydrogen proton transfer between the oxygens attached to carbon1 and carbon2. (see drawings in JH notes 8/02/2020 to 8/03/2020). Glutamine-165 and Histidine-95 are the residues that make this happen. If the glutamine residue is replaced by similar Asparagine, reaction performs 1000x worse. Note that Glu and Asp sidechains are exactly identical *except*  that Asp has just one more CH<sub>2</sub>  in its chain.
		2. There is a lid that keeps the triose phosphate reactant/intermediate/product safe from hydrophilic attack by H<sub>2</sub> molecules. Without this lid, the reaction would perform 100,000x worse. 
		3. Lysine at position 12 is used to create hydrogen bonds that connect to the various oxygens of triose phoshate to stabilize the molecule as the reaciton proceeds. If we use arginine instead of lysine, the reaction performs 200x less efficiently.
	* "Burglar tools" used by influenza virus to escape the lysosome acidic pH trap.

## Lecture 5: Glycolysis and pathways
### August 3-6, 2020
* Biochemical pathways in fermentation
* How does yeast change a hexose sugar (C<sub>6</sub>H<sub>12</sub>O<sub>6</sub>) into CO<sub>2</sub> and alcohol? Multi-step pathway
* Energetics of a single reaction
	* G3P &#8594; DHAP. Lower free energy in this reaction. Gibbs free energy change is about -1.86 kcal/mol. **&#916;G<sup>0</sup>'** pronounced "delta G zero prime"
		* The activation barrier is very high.
		* But even there is enough TIM, because of entropy and relative concentration of product/reactant, there will always be *some* reactant left at equilibrium
		* The way to understand whether the forward or reverse reaction is favored is based on delta G  **&#916;G** (distinct from &#916;G<sup>0</sup>' above).
		* See also handwritten JH notes 8/03/2020
	* Tricks to make a reaction move forward even when &#916;G is positive (favoring reverse rxn)
		* Trick #1: Direct coupling
			* Couple 2 reactions together. Aka even if A &#8594; B is unfavorable, if C &#8594; D is extremely favorable, then A+C &#8594; B+D might move forward. Example, pair reaction with ATP
			* Convert ATP to ADP (adenosine diphosphate). ATP &#8594; ADP has a &#916;G = -7.3 kcal/mol
		* Trick #2: Indirect coupling
			* Have a "next step" reaction, e.g., B &#8594; C which is highly favorable.
			* So again, assume A &#8594; B has positive &#916;G (and is not favorable in forward direction).
			* Assume B &#8594; C has a very negative &#916;G (and is very favorable in forward direction)
			* Then the overall reaction A &#8594; B &#8594; C will move forward b/c the 2nd rxn constantly removes B, lowering the concentration of [B], and thereby encouraging forward progress of A &#8594; B
* Intro to glycolysis: the breaking of sugar. See JH handwritten notes and textbook references:
	* **Alberts** 
		* Chapter 2: p.80-83: ATP and NAD<sup>+</sup>/NADH. See detailed 10 step version of glycolysis in Panel 2-8 on p. 120-121. **See also JH handwritten notes of this 10-step process August 7-9, 2020** 
		* Chapter 14: Mitochondria and Chloroplasts. p. 817 Citric Acid Cycle (fka Krebs Cycle). p. 819 diagram of overall citric acid cycle, Figure 14-10
	* **Sadava** Chapter 9, p.165 - 182
		* p. 167: Redox reactions, diagram of methane <--> methanol <--> formaldehyde <--> Formic acid <--> carbon dioxide. **For best version of Jeff's redox diagram, see JH notes 8/06/2020**
		* p. 168: summary diagram of glucose &#8594; glycolysis &#8594; pyruvate &#8594; branch between aerobic respiration and anaerobic fermentation
		* p. 177: more detailed summary of reactants and products for glucose &#8594; glycolysis &#8594; pyruvate &#8594; branch between aerobic respiration and anaerobic fermentation
		* p. 169: detailed chemical diagram from input glucose to output pyruvate
		* p. 170: diagram of citric acid cycle
* Regulation of pathways. Negative feedback loops aka balancing loops.
	* [Allosteric regulation](https://en.wikipedia.org/wiki/Allosteric_regulation) changes the shape of the enzyme to control it's activity. Thus, let A &#8594; B &#8594; C &#8594; D &#8594; E. If/when [E] is too high we want E to act as a regulator to change the performance of the C&#8594;D enzyme, we could have the enzyme have a different location ("allo" Greek root for different/other) binding site on the enzyme. And E attaches to this binding site, changing the enzyme shape and reducing activity of the C &#8594; D reaction. aka **feedback inhibition**
* In contrast, **feedforward activation** if there is too much reactant and not enough product and we want the mechanism to work faster. This upregulates the activity of the relevant enzymes
* Cellular respiration and fermentation

### Handwritten notes on 10 steps of glycolysis and NAD<sup>+</sup> &#8594; NADH
* Drawn from Alberts Chapter 2: p.80-83: ATP and NAD<sup>+</sup>/NADH. See detailed 10 step version of glycolysis in Panel 2-8 on p. 120-121. 
* Also see this page on [glycolysis](/glycolysis/)
* For detailed drawings of [NAD (Nictonamide Adenine Dinucleotide)](https://en.wikipedia.org/wiki/Nicotinamide_adenine_dinucleotide) in both oxidized NAD<sup>+</sup> and reduced NADH forms, refer to JH handwritten notes August 7-9, 2020.
* Note also that [pure adenine](https://en.wikipedia.org/wiki/Adenine) is just the nucleobase by itself. [Adenosine](https://en.wikipedia.org/wiki/Adenosine) refers to adenine covalently bonded to a ribose sugar like in [ATP](https://en.wikipedia.org/wiki/Adenosine_triphosphate), [NAD](https://en.wikipedia.org/wiki/Nicotinamide_adenine_dinucleotide), or [RNA](https://en.wikipedia.org/wiki/RNA).

## Lecture 6: Intro Mendel and TH Morgan Fruitfly experiments
### August 6-15, 2020
* Biography of Mendel, initial pea experiments
* Definitions of genetic terms
	* Gene: discrete factors of inheritance
	* Allele: Alternative versions of a gene, e.g., A vs. a
	* Genotype: The two alleles carried by an organism
	* Homozygote: Two of the same allele. e.g., AA or aa
	* Heterozygote: Two different alleles in a genotype, e.g., Aa
	* Phenotype: External appearance or trait. 
* Important distinction. There are **no such things as dominant and recessive alleles**. ONLY dominant versus recessive phenotypic traits. Especially important because the same gene can contribute to multiple traits, some of which are dominant and some of which are recessive.
* Error in video lecture. Prof. Lander says that green is dominant to yellow color in pea pants. It's actually the reverse; yellow pea color is dominant vs. green peas
* Mendel's Second Law:  Independent Assortment
* Circa 1890, cytology led to discovery of these mysterious "colored organelles" or colored bodies which were named "chromosomes". Organization of chromoses into bundles used during cell division aka mitosis.
* Description of Mitosis and Meiosis
* Coincidence that  Mendel luckily only studied 7 traits in peas. and peas only happen to have 7 chromosomes???

## Lecture 7: Meiosis
### August 16-18, 2020
* During meiosis, homologs
* Review of Mendelian inheritance versus chromosomal inheritance
* Fruit flies and linkage among chromosomes. Thomas Hunt Morgan was a great skeptic and did not trust that people kept inventing "inheritance factors" just to explain new data  a posteri. Should have an a priori model that is then properly tested
* Pictures of fruitflies with various traits. "+" = "wildtype" phenotype = "wt"
* See Handwritten notes for multiple generations of wildtype versus mutant flies for Body phenotype: (either dominant **wt** or recessive **black body**) and for Wing phenotype: (either **wt** or recessive **vestigial wing**)
* Experimentally, TH Morgan found among the children of Parent 1 ( wt  b * wt vg) crossed with Parent 2 (b b * vg vg) these children: 
	* **965 identical to parent 1** (wt b * wt vg = heterozygous for both Body and Wing)
	* **944 identical to parent 2** (bb * vg vg = homozygous recessive for Body, homozygous recessive for Wing)
	* **206 new recombinant fly** (wt b * vg vg = heterozygous for Body and homozygous recessive vestigial wing for Wing)
	* **105 new recombinant fly** ( b b * wt vg = homozygous for recessive black Body and heterozygous for Wing)

### Linkage Maps

* In 1911, 19-year-old Columbia undergraduate [Alfred H. Sturtevant](https://en.wikipedia.org/wiki/Alfred_Sturtevant) took the *frequency of recombination* ratio data home from the Morgan fruit fly lab and created the [first genetic linkage maps](https://en.wikipedia.org/wiki/Genetic_linkage).
* "The recombination frequency equals the number of recombinants divided by the total progeny times 100. A longer distance between two genes increases the chance of crossover between two genes. A recombination frequency between 0 and 50 indicates that the genes are located on the same chromosome. Genes that are far apart on the same chromosome or lie on different chromosomes show a recombination frequency of 50%. Since the most extreme case of un-linkage is independent assortment, which gives 50% recombinant offspring, the maximum value for the recombination frequency is 50%."
* the recessive traits black body and vestigial wing assort together about 17% of the time.
* Sturtevant is doing "pairwise" aka 2-factor crosses for 2 different genes
* Further, let's consider a **3-factor cross**. How many gametes are possible when 2 parents mate when one considers three phenotopic traits (aka 3 genes)? Answer is 8 = 2<sup>3</sup>.

### Linkage Mapping
* Sample drosophila mutations: [Antennapedia](https://en.wikipedia.org/wiki/Antennapedia) and [Double Wing aka Ultrabithorax](https://en.wikipedia.org/wiki/Ultrabithorax)
 * Test question on Linkage Mapping: "The maximum recombination frequency seen between any two genes on the same chromosome is about 50%. Why this is the case?"
	* *Answer:* Imagine a case where the two genes are located on different chromosomes, so that during meiosis the segregation of alleles of one gene is completely independent of the alleles of the other gene (this is stated in Mendel's Second Law and is known as the law of independent assortment). On average, you would see the parental arrangement 50% of the time. When the two genes are far apart on the same chromosome, there can be 1, 2, 3, or more recombination events between the two genes. In this case, it is random whether there will be an even or odd number of recombination events, so on average, you would see the parental arrangement 50% of the time.
 
* **Sex-linked traits** See notes on Mac bionotes directory including *.png screenshot. 

 
### Meiosis review by PhD student Michelle
 * Diploid number refers to the number of total chromosomes = 2n where n refers to the number of pairs. e.g., human beings have n = 23 for **23 pairs of chromosomes**. So humans diploid number = 2n = 2 * 23 = **46 chromosomes** where each chromosome has a **homologous** "semi-duplicate" chromosome with the same genes but possibly different alleles at each gene loci. 
 * See also supplemental [videos](https://www.youtube.com/watch?v=dLy_GL5ajco&feature=youtu.be) about Punnett Squares, etc.  
 

 
## Lecture 8: Basics of Human Genetics
### August 17-18, 2020

### X-linked Recessive Inheritance. 
 * All men who receive this mutation express it. Women can be carriers or only express trait if they are homozygous recessive on both X chromosomes. Examples: red-green color blindness, hemophilia. Rules include:
	 * Predominantly affects males because women are only affected when they inherit allele on both of their X chromosomes.
	 * Affected men do not pass this trait on to sons unless man mates with a woman who has the recessive allele. (b/c sons will always get their X chromosome from their mom.) This characteristic is what led to the idea of traits "skipping a generation"
	 * A woman who is a carrier who has sons will see this trait show up in 50% of her sons.
	 * A woman who is affected by this trait is homozygous recessive (+/+) will pass this trait onto 100% of her sons.
 * Population genetics. Relationship between phenotype and mutant allele. E.g., red-green color blindness occurs in 8% of the male popuation. This means that 8% of all X chromosomes have this allele. Allele frequence = 8%.

### Autosomal dominant inheritance
 * Autosome = chromosomes that are not sex chromosomes. Humans have 22 pairs of autosomes, 1 pair of sex chromosomes.
 * Autosomal dominant inherited traits. Example: [Huntington's disease](https://en.wikipedia.org/wiki/Huntington%27s_disease). Population frequency is quite low: ~5 in 100,000 people have Huntington's. 
 * Another example: Familial hypercholesterolemia aka FH affects 1 in 500 people. Using a Punnett Square / [Hardy-Weinberg](https://en.wikipedia.org/wiki/Hardy–Weinberg_principle) quadratic equation, we can derive that this allele will occur in roughly 2q aka ~1 in 1000 chromosomes
 * Rules
	 * 50% of offspring are affected
	 * Affects all genders equally
	 * Unaffected never transmits
	 * Never "skips a generation"
	 * If a parent is homozygous for this mutant dominant allele, that means that 100% of their offspring will have express this trait.
 * Breast cancer for example of penetrance and sex-limited traits.
	 * Full penetrance of a dominant trait. Huntington's and FH are full penetrance.There can be incomplete penetrance of a dominant trait. Breast cancer in the form of both BRCA1 (chromosome 17) and BRCA2 (chromosome 13) are examples of incomplete penetrance. About 60% of the women who have either allele will develop breast cancer (not 100% which we would expect for an automsomal domaninant full penetrance type gene).
	 * Also, males still exhibit some incidence of breast cancer, at 1% for male carriers of BRCA1 and 6.5% for male carriers of BRCA2. This is a *sex-limited* disease.
 
### Autosomal recessive inheritance
* Rules: (1) Trait does not appear in every generation (illusion of "skipping a generation"). (2) Parents of affected children generally are not affected.
* Example of cystic fibrosis. About 1 in 2000 chromosomes have this mutant allele which produces the recessive trait of CF. aka q<sup>2</sup> = 1/2000. So the % of the population that has CF is q= square root (1/2000) aka q = about 1/44. about 1/22 of people are carriers for CF allele on chromosome 7.
* Predicted Mendelian ratio is that two carrier parents should have 1/4 of their offspring exhibit CF. However, the observed ratio was 1/3. The reason was because humans have small families and so there was an *undercounting* of families that should include people with CF allele. If humans typicially had 1000 children per family, the expected ratio of 1/4 would probably be observed.


### How Real Human Genetics differs from the simple rules in intro biology textbooks
* Example of [Charles Davenport](https://en.wikipedia.org/wiki/Charles_Davenport) who produced a [pedigree chart](https://www.khanacademy.org/science/high-school-biology/hs-classical-genetics/hs-pedigrees/a/hs-pedigrees-review) that claimed that the tendency of people to become sea captains conformed to Mendelian genetics. This was called [thalassophilia](https://dnalc.cshl.edu/view/15914-Thalassophilia-pedigree.html).
* Example of early 20th-century scientists who thought that [pellagra](https://en.wikipedia.org/wiki/Pellagra) was inherited rather than caused by poverty and malnutrition.
* Important to be careful!

### Archibald Garrod and Alkaptonuria
* In early 20th century London. [Alkaptonuria](https://en.wikipedia.org/wiki/Alkaptonuria) = "black urine" in infants. In 8 of the 17 cases, the affected children were the result of first cousin marriages. If only 1 grandparent has a single Alkaptonuria allele (which expresses a recessive trait), and assuming that the spouses who marry into the family in the F1 generation are 100% wildtype, and assuming that first cousins in the F2 generation intermarry, then there is a 1/16 chance that the great-grandchildren born in the F3 generation will be homozygous for the alkap allele and express the recessive trait.
* Next Dr. Garrod studied the chemical compound in the urine that turns things black in air is [homogentisic acid (HGA)](https://en.wikipedia.org/wiki/Homogentisic_acid). The oxidation of HGA creates this black color. The phenol ring looks like amino acids phenylalanine and tyrosine. Experimentally, when Garrod fed these babies extra protein, extra phenylalanine, and extra tyrosine, more HGA was produced.
* He knows that in  wildtype human digestion, protein is broken down into &#8594; amino acids (like Phe, Tyr) &#8594; homogentisic acid (HGA) &#8594; further broken down into something else. In 1908, he gives a lecture claiming that babies with alkaptonuria are missing the enzyme for that final step so they simply accumulate more and more HGA that must be urinated out. Wildtype babies have something (Garrod didn't necessarily know it was an enzyme), that breaks down HGA.

### Josh Meisel and Pedigree Analysis
* Meisel, C. elegans and infectious bacteria PhD advised by [Dennis Kim](http://kimlab.mit.edu/people.html) and  current postdoc in [Vamsi Mootha's mitochondria lab](http://mootha.med.harvard.edu/members.html) at Harvard Medical School. 
 
 
## Lecture 9: Genetic basis of biochemistry
### August 19-20, 2020 
*  [George Beadle](https://en.wikipedia.org/wiki/George_Beadle) and [Edward Tatum](https://en.wikipedia.org/wiki/Edward_Tatum) work. After a lot of frustrating work with the chimeric eyes of mutant drosphila, they switched to a model organism [Neurospora](https://en.wikipedia.org/wiki/Neurospora), single celled fungus.
* For purposes of today's lecture, Lander will instead talk about Saccharomyces cerevisiae ([S.cerevisiae](https://en.wikipedia.org/wiki/Saccharomyces_cerevisiae)) common baker's yeast.
* S. cerevisiae are true eukaryotes. 2n = 16 pairs of chromosomes. n = 32 total chromosomes, counting homologous.
	* Diploid cells undergo mitosis.
	* Also, yeast diploid cells (with 2n chromosomes each) can undergo meiosis to create haploid gametes (with n chromosomes each) that can mate like humans, producing F1 cells with diploid chromosomes again.
	* Finally (unlike humans), yeast gametes can themselves undergo mitosis without need for mating etc.
* Auxotrophs vs. prototrophs
* Esther Lederberg makeup compact to do replica plating. Much more efficient than individual scrapings of yeast colonies with a toothpick
* Prefer to screen potential mutants using haploid cells because they can determine a mutant allele whether it is dominant or recessive. Using diploid yeast would only pick up dominant allele.
* Haploinsufficiency
* Characterizing mutants test #1: Dominant or Recessive?
	* Simple cross with a wildtype haploid through mating.
*  Test #2: Find complementation groups
	* Definition of [complementation](https://bio.libretexts.org/Bookshelves/Genetics/Book%3A_Working_with_Molecular_Genetics_(Hardison)/Unit_I%3A_Genes%2C_Nucleic_Acids%2C_Genomes_and_Chromosomes/1%3A_Fundamental_Properties_of_Genes/1.4%3A_Complementation_and_Recombination): "the ability of two mutants in combination to restore a wildtype phenotype". Meanwhile, "dominance in a heterozygote indicates the ability of a wildtype allel to complement loss-of-function alleles.
	* Start with 50 different mutants of yeast; none of them can produce arginine. They are all called "arginine [auxotrophs](https://en.wikipedia.org/wiki/Auxotrophy)". 
	* Cross some of them to clarify if these mutants are based on one or multiple genes.
	* Doubly heterozygous aka "double het". 
	* Gene that makes arginine is called arg1 which may have various mutations: m1, m2, m3.
	* A second gene that contributes to the production of arginine is called arg2
	* Failure to complement means that two mutants are on the same gene. B/c a successful complementation means that one of them is a dominant allele on the same gene. 
* Test #3: [Epistasis](https://en.wikipedia.org/wiki/Epistasis) Test
	* Mutations in a sequential biochemical pathway to indicate which enzymes are affected.
	* Interesting [history](https://en.wikipedia.org/wiki/Epistasis#History) about how the idea of epistasis has evolved from the early days of genetics in the first decades of the 20th century.
* Ryan, PhD video on lab techniques with S. cerevisiae
 
## Lecture 10: Intro to DNA Structure
### August 20-22, 2020 
* [Frederick Griffith's](https://en.wikipedia.org/wiki/Frederick_Griffith) search for the "S-Transforming Principle". 
	* Griffith's [S. pneumoniae bacteria](https://en.wikipedia.org/wiki/Streptococcus_pneumoniae) experiments with [rough vs smooth bacteria](https://en.wikipedia.org/wiki/Griffith%27s_experiment) infecting mice.
	* "The Transforming Principle" aka alchemical substance.  
	* Pnuemococcus caused 50% of the deaths in the Great Influenza of 1918-1920
* [Oswald Avery, Colin MacCleod, and Maclyn McCarty experiments](https://en.wikipedia.org/wiki/Avery–MacLeod–McCarty_experiment)
	* Through repeated purification of the S-transforming principles, eliminated protein, eliminated proteins, etc. All they are left with is DNA. In 1943, publish the paper claiming that DNA is the hereditary material. People don't believe it b/c reviewers worry that there was protein contamination b/c DNA seemed to be too simple to be the hereditary material.
* Structure of DNA
	* Pentose deoxyribose is missing a hydroxyl at the 2'-carbon.
	* Bases. Purines (A, G) have 2 carbon rings. Pyramidines (T, C) have 1 carbon ring. These bases are connected to the 1'-carbon of deoxyribose
* [Al Hershey and Barbara Chase experiments](https://en.wikipedia.org/wiki/Hershey–Chase_experiment) on E. coli bacteriophage viruses. 
	* Uniquely tagged protein with radioactive isotopes of sulfur. Since DNA doesn't have sulfur, DNA will *not* be radioactively tagged.
	* Alternately, DNA has phosphorus atoms but if you choose the right proteins, amino acids do not.
* Now, there are two converging lines of evidence that DNA is the genetic material: Avery-MacCleod-MacCarty experiments and Hershey-Chase experiments.
* [Erwin Chargaff](https://en.wikipedia.org/wiki/Erwin_Chargaff) did a lot of experiments and formulated [Chargaff's rules](https://en.wikipedia.org/wiki/Chargaff's_rules), which indicates that the number of A = T and C = G regardless of what species examined.
* Linus Pauling's proposed triple axis model for DNA structure has phosphates in the center axis which can't work b/c they are all strongly negative and will repel each other.
* Supplemental vidoes. 
	* dsDNA = double-stranded aka the most common formation <a href="https://microbenotes.com/different-forms-of-dna-b-form-a-form-z-form/">B-form DNA</a>
	* Shirleen PhD student. Replication occurs at [origin of replication](https://en.wikipedia.org/wiki/Origin_of_replication) on the DNA strand. Enzymes involved: helicase, topoisomerase, DNApolymerase, primase, RNAseH, ligase. Okazaki framgnets

## Lecture 11: Central Dogma 1 - DNA Replication
### August 22-25, 2020
* Matthew Meselson and Franklin Stahl experiments proving the [semi-conservative nature of DNA replication](https://en.wikipedia.org/wiki/Meselson–Stahl_experiment). Used regular N14 nitrogen and heavy nitrogen N15 to specifically label the strands of DNA, proving that replication is semi-conservative

### Basic Replication
* DNA Polymerase
	* 1956, Arthur Kornberg manually adds radioactively tagged dATP, dCTP, dGTP, and dTTP. Short for dexoyadenosine triphosphate (dATP), deoxycytidine triphosphate (dCTP), deoxyguanosine triphosphate (dGTP), and deoxythymidine triphosphate (dTTP). See table on [this wiki page](https://en.wikipedia.org/wiki/Nucleoside#Use_in_medicine_and_technology).
	* Explanation of why DNA must grow in the 5' &#8594; 3' direction; otherwise, the last monomer on the growing chain might hydrolyze
* Other enzymes involved:
	* **Helicase** unwinds helix
	* **Primase** creates the RNA primer to get the replication started
	* **DNA Polymerase** attaches the the RNA primer and adds new monomeric nuclueotides to the growing DNA polymer strand (for both leading and lagging strands of the replication fork)
	* As replication proceeds, there is greater and greater tension ahead of the replication fork. **Topoisomerase** releases this tension by making strategic cuts in the parent DNA double strand.
	* **Ligase** "glues up" and seals the separated Okazaki fragments on the lagging strand by forming phosphodiester bonds betwen the 3' OH and the 5' phosphate group adjacent fragments.

### Methods that ensure replication fidelity
* DNA Polymerase does proofreading.
* Exonuclease runs in the 3'&#8594;5' direction and takes bases off of the chain (opposite activity compared to DNA Polymerase). Improves accuracy from 1/1000 to 1/1,000,000. One error in every 10<sup>3</sup> becomes after exonuclease one error in every million 10<sup>6</sup>.
* Another method: mismatch detection and repair. When a basepair shows a mismatch between two nucleotides, how does E.coli know which one is the "correct" nucleotide and which is the new "wrong" nucleotide? Luckly, there is a methylation process that marks the original parent strand of DNA. New, daugher strands of DNA lack these methylization markers for a bit. So E.Coli only repairs the unmethylated "newborn" DNA strand.
* After combining proofreading, exonuclease, and mismatch repair, error rate becomes about one in a billion bp (aka 1 in 10<sup>8</sup>). Given that the human genome is only about 3 billion bp, this means that there are only a total of 30-40 errors per replication.
* The real workhorse for replication is Pol III, not Pol I and Pol II that were discovered earlier.  Pol I is not the "real polymerase" and it polymerizes at only 10-20 nucleotides/second. It runs out of steam after about 50 nucleotides. Discovered DNA Polymerase II in a mutant e. coli that could still go through mitosis even though this mutant did not produce DNA Poly I.

### More details on Replicon
* For much more detail about DNA Replication, see handwritten notes and [notes on Molecular Biology of the Cell, Alberts](/alberts-5e) Chapter 5.
	* In particular, refer to the [wiki article for the replisome](https://en.wikipedia.org/wiki/Replisome), this [2012 diagram](https://bmcbiol.biomedcentral.com/articles/10.1186/1741-7007-10-34/figures/1), and JH handwritten notes from 9/06/2020


## Lecture 12: Central Dogma 2 - DNA Transcription and Translation
### September 4, 2020
* Uracil is a pyrimidine that is exactly like thymine except it is missing the -CH<sub>3</sub> methyl group off the 5 carbon
* Because DNA transcription process does not use error-correction / proofreading mechanisms like replication, accuracy for RNA creation is only 1 error in 10<sup>4</sup> (10,000) vs the 1 error in 10<sup>9</sup> (1 billion) for DNA replication.

## Lecture 13: Central Dogma 3 - Variations from the Dogma
### September 7, 2020
* How do replication, transcription, and translation work in eukaryotes, prokaryotes, and viruses?

### Replication
* Viral replication (5 - 200 kb) and has these types of genetic material in their protein capsid:
	* double stranded linear DNA
	* double stranded circular DNA
	* single stranded DNA
	* single stranded RNA in both [+ssRNA aka positive strand](https://en.wikipedia.org/wiki/Positive-strand_RNA_virus) and [-ssRNA aka negative strand](https://en.wikipedia.org/wiki/Negative-strand_RNA_virus) varieties. Retroviruses like HIV use reverse transcriptase to insert themselves into host DNA.
* Prokaryotes (700 - 6,000 kb. e.g., *E. coli* has 4,000 kb genome)
	* Replication in prokaryotes is pretty straightforward b/c it has 1 large chromosome that is circular. Easy replication for various bacterial plasmids as well.
* Eukaryotic replication (12 Mb - 4,000 Mb; see [PSU notes, Lecture 2](https://www.bionotes.org/psu334/))
	* However, replication in eukaryotes is more complicated because eukaryotic chromosomes are linear. 
	* This is fine for leading strand replication but lagging strand synthesis will screw up unless the final Okazaki fragment lines up exactly with the length of the chromosome. 
	* Instead, there are *telomeres* on the end of the each chromosomes, consisting of repeated sequences of **5'-TTAGGG-3'** (aka "T2AG3"). These **T2AG3** sequences are elongated by [telomerase](https://en.wikipedia.org/wiki/Telomerase).
	* Note that there are different repeating sequences depending on genera. See [table at wiki article here](https://en.wikipedia.org/wiki/Telomere#Sequences).

### Transcription
* Prokaryote transcription is pretty straightforward
* Transcription in eukaryotes. The [immature RNA aka primary transcript](https://en.wikipedia.org/wiki/Primary_transcript) is processed into [mature mRNA.Three](https://en.wikipedia.org/wiki/Mature_messenger_RNA). Three things happen during this processing:
	* A single GTP (aka  Guanine with three phosphates) is added as a "start" cap to the 5' end of mature mRNA in a process called "[capping](https://en.wikipedia.org/wiki/Five-prime_cap)".
	* The 3' end of the mRNA goes through [polyadenation](https://en.wikipedia.org/wiki/Polyadenylation). In other words, a **poly(A) tail** (*aka* a repetitive series of adenines, AAAA...) are added to the 3' end. Virtually all eukaryotic mRNA's have poly(A) tails.
	* [Introns](https://en.wikipedia.org/wiki/Intron) removed by RNA splicing. Alternate splicing means that the same DNA gene and same immature mRNA can be spliced into different mature mRNAs (with different exons) that therefore express different proteins. In other words, most eukaryotic genes are called ["split" or "interrupted"](https://en.wikipedia.org/wiki/Interrupted_gene).
		* Intron and exon are terms coined by [Walter Gilbert](https://en.wikipedia.org/wiki/Walter_Gilbert) at Harvard; Glibert also co-founded Biogen and Myriad Genetics and, in 1986, first proposed the [RNA World hypothesis](https://en.wikipedia.org/wiki/RNA_world).
		* The existence of interrupted genes and RNA splicing was discovered by at Cold Spring Harbor Lab by [Richard Roberts](https://en.wikipedia.org/wiki/Richard_J._Roberts) and [Phillip Sharp](https://en.wikipedia.org/wiki/Phillip_Allen_Sharp). Sharp would later take a leadership role at the MIT Biology Dept and co-found Biogen, Alnylam Pharmaceuticals, and Magen Biosciences.

## Lecture 14: A Tale of Two Genes– &#946;-Galactosidase and &#946;-Globin

### September 9, 2020

 
 
---		

# Checklist
### <del>Unit 1: Introduction and the Biochemistry of Life</del>
<del>Lectures 1, 2, 3</del>
### <del>Unit 2: Biochemistry: Proteins, Enzymes, Glycolysis</del>
<del>Lectures 4, 5</del>

### <del>Unit 3: Genetics including meiosis</del>
* <del> Lectures 6, 7 *all completed including Michelle's PhD video on meiosis and [Barbara McClintock's work on corn recombination](https://en.wikipedia.org/wiki/Barbara_McClintock)*</del>
* **Finish problem set for Unit 3: Genetics I**

### <del>Unit 4: Human and Biochemical Genetics</del>
* <del>Lectures 8, 9
* Watch lecture 8 final video clip of Meisel PhD video on pedigree analysis
* **Finish problem set for Unit 4: Genetics II**

### <del> Unit 5 : Molecular Biology I - DNA Replication</del>
* <del>Lecture 10,11</del>

### Unit 6 : Molecular Biology II - Transciption, Translation, Variations
* <del>Lecture 12: transcription and translation</del>
* **Lecture 13 on variations between viruses, prokaryotes and eukaryotes**
* Lecture 14:A Tale of Two Genes– &#946;-Galactosidase and &#946;-Globin

### Unit 7: Recombinant DNA
Lectures 15, 16, 17
### Unit 8: Genomics I - Human Genome
Lectures 18, 19
### Unit 9: Genomics II - Observing and Perturbing the Genome to Probe Function
Lectures 20, 21
### Unit 10: Disease, Science, and Society
Lectures 22, 23, 24

---

# Miscellaneous notes

* Good [YouTube video](https://youtu.be/ZesFyuFj30s) explaining how to use free online MolView.org. 
* Excellent [before and after video](https://twitter.com/DrewCoffman/status/1274743473732632576) showing off the special effects a single artist can create in Blender. Higher res version on [Youtube](https://youtu.be/FFJ_THGj72U)
* [Wikipedia article](https://en.wikipedia.org/wiki/Drew_Berry) about Australian science animator [Drew Berry](https://www.macfound.org/fellows/27/)
	* Read the comment thread at [this io9 interview piece](https://io9.gizmodo.com/animator-drew-berry-is-here-to-talk-to-you-about-making-1644346171)
	* [Good FAQ](https://www.drewberry.com/faq) about tools and getting started.
